css-three-column-layout/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Three Column Layout</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="headline">
+        <h1>
+          Designed for life science. Optimized for SaMD and Designed Health.
+        </h1>
+      </div>
+      <div class="content">
+        <div class="nestedcontent">
+          <span class="image">
+            <img src="images/merge.png"/ alt="Internet Merge">
+          </span>
+          <h3>Streamlined QMS</h3>
+          <p>
+            A QMS that includes modlules for all stages of the product life
+            cycle from Design Control to CAPA
+          </p>
+        </div>
+        <div class="nestedcontent">
+          <span class="image">
+            <img src="images/cloud-computing.png"/ alt="Internet Merge">
+          </span>
+          <h3>Integrates with existing tools</h3>
+          <p>
+            Enzyme QMS Integrates with your existing tools, such as JIRA,
+            GitHub, and Jenkins with Enzyme
+          </p>
+        </div>
+        <div class="nestedcontent">
+          <span class="image">
+            <img src="images/discussion.png"/ alt="Internet Merge">
```

Comment:
You have a `/` after the quote that's closing off the tag too early so your `alt` is getting cut off. Delete the `/`